packages/boot/src/booters/repository.booter.ts

```diff
@@ -35,7 +36,7 @@ export class RepositoryBooter extends BaseArtifactBooter {
      * accordingly.
      */
     // tslint:disable-next-line:no-any
-    if (!(this.app as any).repository) {
+    if (!(this.app as AppWithRepository).repository) {
```

Comment:
One more idea - maybe we should change constructor to expect AppWithRepository instead of Application? That will allow us to avoid all `as AppWithRepository` casts in this file.